src/libs/actions/Report.js

```diff
@@ -2081,7 +2115,10 @@ function leaveRoom(reportID, isWorkspaceMemberLeavingWorkspaceRoom = false) {
         },
     );
 
-    if (isWorkspaceMemberLeavingWorkspaceRoom) {
+    const lastAccessedReportID = updateRecentlyAccessedReportID(reportID, true);
+    if (lastAccessedReportID) {
+        Navigation.dismissModal(lastAccessedReportID);
+    } else {
```

Comment:
@eh2077 could you explain this? i am not sure if i understood this correctly.
we did we remove the `isWorkspaceMemberLeavingWorkspaceRoom` from the condition? shouldn't this be something like.

```js
if(lastAccessedReportID && isWorkspaceMemberLeavingWorkspaceRoom) {
} else if(isWorkspaceMemberLeavingWorkspaceRoom) {}
```

Reply:
@getusha Good question! I just followed 
> Drop users into whatever chat they were previously viewing before the viewed the room or group they left. Dropping a user into a chat they were just viewing is more likely to be useful than dropping a user into a Concierge DM.

So, as long as `lastAccessedReportID` is not empty, then navigate to the chat. Or else open the concierge chat. 

@techievivek Is it correct?

Reply:
Yeah, as long as we have lastAccessedReportID non-empty(no matter what chat report they just left).